src/DayColumn.js

```diff
@@ -142,10 +142,8 @@ class DayColumn extends React.Component {
       , showMultiDayTimes
       , culture
       , eventPropGetter
-      , selected
+      , selected, eventTimeRangeFormat, components: { eventComponent }
       , messages
-      , eventComponent
-      , eventTimeRangeFormat
```

Comment:
Seems like a rebate error

Reply:
Actually this is because of a fix I made in #406. eventComponent was being destructured incorrectly in the DayColumn component's renderEvents function (so it was always undefined). We need to be able to specify custom eventComponent for the addOn to work properly.. But its possible this has since been fixed some other way.

Reply:
Instead of using event component. I updated the addon to pass the ResizeEvent to `components: { day: { event }, week:  { event } }` as opposed to creating a new prop inside of components. I think this is how the library is intended to be used.